docker/config/app.env

```diff
@@ -10,4 +10,6 @@ REDIS_URI=redis://redis:6379/0
 # DB
 DB_READONLY_URI=mysql+pymysql://root:location@mysql:3306/location
 DB_READWRITE_URI=mysql+pymysql://root:location@mysql:3306/location
-DB_DDL_URI=mysql+pymysql://root:location@mysql:3306/location
+
+# DB url for running alembic migrations
+SQLALCHEMY_URL=mysql+pymysql://root:location@mysql:3306/location
```

Comment:
I named this closer to what alembic calls it figuring maybe that makes it clearer that it's an alembic thing and not an app configuration thing.

It is in `app.env`. I'm kind of loathe to create another env file, though. I'm not sure how to make this clearer without making it more complex. Maybe we rename `app.env` to `local_dev.env`?

Reply:
I'm going to switch it. It represents an environment (e.g. local dev environment, stage, prod)--not a service (e.g. webapp, celery).